l1-contracts/test/Decoder.t.sol

```diff
@@ -258,17 +255,6 @@ contract DecoderTest is Test {
     assertEq(logsHash, referenceLogsHashFromIteration3, "Incorrect logs hash");
   }
 
-  // Tests https://github.com/AztecProtocol/aztec-packages/issues/730 is handled correctly
-  function testLogsDontGetInterpretedAsMessages() public {
-    (,,,,, bytes32[] memory l1ToL2Msgs) = helper.decode(block_with_no_l1_l2_msgs_and_with_logs);
-
-    // Check all l1ToL2Msgs are empty
-    bytes32 emptyMsg = bytes32(0);
-    for (uint256 i = 0; i < l1ToL2Msgs.length; i++) {
-      assertEq(l1ToL2Msgs[i], emptyMsg, "l1ToL2Msgs should be empty");
-    }
-  }
```

Comment:
Why is this test no longer needed?

Reply:
I wrote the test a while ago when I was demonstrating the issue in the test comment. I deleted it now because otherwise I would have to manually update the block value and I just felt like it's not worth the hassle.